apps/website/src/routes/docs/headless/(components)/navigation-bar/index.tsx

```diff
@@ -9,12 +9,12 @@ export default component$(() => {
     <>
       <h2>This is the documentation for the Navigation Bar</h2>
 
-      <div class="flex flex-col gap-8 mt-4">
+      <div class="mt-4 flex flex-col gap-8">
         <h2>Basic Example</h2>
 
-        <NavigationBar class="items-center navigation-bar flex w-full bg-base-100 rounded-lg">
+        <NavigationBar class="navigation-bar bg-base-100 flex w-full items-center rounded-lg">
           <div class="navigation-bar__start flex">
-            <a class="btn btn-ghost normal-case text-xl ">tailwindUI</a>
+            <a class="btn btn-ghost text-xl normal-case ">tailwindUI</a>
```

Comment:
```suggestion
            <a href="" class="btn btn-ghost text-xl normal-case ">tailwindUI</a>
```

let's add an href here so we don't get a warning from qwik